src/util/numeric.h

```diff
@@ -144,36 +125,17 @@ inline bool isInArea(v3s16 p, v3s16 d)
 	);
 }
 
-#define rangelim(d, min, max) ((d) < (min) ? (min) : ((d)>(max)?(max):(d)))
-#define myfloor(x) ((x) > 0.0 ? (int)(x) : (int)(x) - 1)
-
 inline v3s16 arealim(v3s16 p, s16 d)
 {
-	if(p.X < 0)
-		p.X = 0;
-	if(p.Y < 0)
-		p.Y = 0;
-	if(p.Z < 0)
-		p.Z = 0;
-	if(p.X > d-1)
-		p.X = d-1;
-	if(p.Y > d-1)
-		p.Y = d-1;
-	if(p.Z > d-1)
-		p.Z = d-1;
+	if (p.X < 0) p.X = 0;
```

Comment:
Why are these on one line? (@est31 I don't think this should be ignored at all; it's bad style).

Also, unless (d - 1) can equal -1 then these should be more optimally arranged.